composer.json

```diff
@@ -8,18 +8,35 @@
         "php": ">=7.2",
```

Comment:
I think it is okay to support `7.2` for now, but we should note that it is "legacy" or something since our CICD will assume minimum 7.3.